06_puppy/kasunfdo/types.go

```diff
@@ -0,0 +1,29 @@
+package main
+
+import (
+	"sync"
+)
+
+// Data structure for puppy
```

Comment:
This is not the proper format for a doc string. See https://blog.golang.org/godoc-documenting-go-code for details.